.github/workflows/end2end-test.yml

```diff
@@ -2,6 +2,7 @@ name: End-to-End Tests
 
 on:
     pull_request:
+        types: [review_requested, ready_for_review]
```

Comment:
With that the test wouldn't run for further pushes on the PR, for example after a review. You'd have to request a review again which may be a workaround but might get a bit spammy. 

Maybe add also `synchronize` but check for `github.event.pull_request.draft` like done [here](https://github.community/t/dont-run-actions-on-draft-pull-requests/16817/16)? This would not directly skip the workflow but still would be faster than running the full workflow.

Reply:
I just tried setting this PR back to draft, pushed something new, and it doesn't seem to stop checks from re-running. A Request for Changes here would be appreciated in order to check that case!

> Maybe add also `synchronize` but check for `github.event.pull_request.draft` like done [here](https://github.community/t/dont-run-actions-on-draft-pull-requests/16817/16)? This would not directly skip the workflow but still would be faster than running the full workflow.

Checking for draft, in my opinion, is not far enough to save us run time, as most PRs are not Drafts (139 out of 687 as of this comment.) I'll add synchronize to keep testing!